lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -2037,433 +2005,124 @@
-        ],
-        "description": "Ensures that lists are structured correctly",
-        "help": "<ul> and <ol> must only directly contain <li>, <script> or <template> elements",
-        "helpUrl": "https://dequeuniversity.com/rules/axe/4.1/list?application=lighthouse",
-        "nodes": []
+        "id": "input-image-alt"
       },
       {
-        "id": "listitem",
-        "impact": null,
-        "tags": [
-          "cat.structure",
-          "wcag2a",
-          "wcag131"
-        ],
-        "description": "Ensures <li> elements are used semantically",
-        "help": "<li> elements must be contained in a <ul> or <ol>",
-        "helpUrl": "https://dequeuniversity.com/rules/axe/4.1/listitem?application=lighthouse",
-        "nodes": []
+        "id": "list"
       },
       {
-        "id": "meta-refresh",
-        "impact": null,
-        "tags": [
-          "cat.time-and-media",
-          "wcag2a",
-          "wcag2aaa",
-          "wcag221",
-          "wcag224",
-          "wcag325"
-        ],
-        "description": "Ensures <meta http-equiv=\"refresh\"> is not used",
-        "help": "Timed refresh must not exist",
-        "helpUrl": "https://dequeuniversity.com/rules/axe/4.1/meta-refresh?application=lighthouse",
-        "nodes": []
+        "id": "listitem"
       },
       {
-        "id": "role-img-alt",
-        "impact": null,
-        "tags": [
-          "cat.text-alternatives",
-          "wcag2a",
-          "wcag111",
-          "section508",
-          "section508.22.a",
-          "ACT"
-        ],
-        "description": "Ensures [role='img'] elements have alternate text",
-        "help": "[role='img'] elements have an alternative text",
-        "helpUrl": "https://dequeuniversity.com/rules/axe/4.1/role-img-alt?application=lighthouse",
-        "nodes": []
+        "id": "meta-refresh"
       },
       {
-        "id": "scrollable-region-focusable",
-        "impact": null,
-        "tags": [
-          "cat.keyboard",
-          "wcag2a",
-          "wcag211"
-        ],
-        "description": "Elements that have scrollable content should be accessible by keyboard",
-        "help": "Ensure that scrollable region has keyboard access",
-        "helpUrl": "https://dequeuniversity.com/rules/axe/4.1/scrollable-region-focusable?application=lighthouse",
-        "nodes": []
+        "id": "role-img-alt"
       },
       {
-        "id": "select-name",
-        "impact": null,
-        "tags": [
-          "cat.forms",
-          "wcag2a",
-          "wcag412",
-          "wcag131",
-          "section508",
-          "section508.22.n",
-          "ACT"
-        ],
-        "description": "Ensures select element has an accessible name",
-        "help": "Select element must have and accessible name",
-        "helpUrl": "https://dequeuniversity.com/rules/axe/4.1/select-name?application=lighthouse",
-        "nodes": []
+        "id": "scrollable-region-focusable"
       },
       {
-        "id": "tabindex",
```

Comment:
all these changes are correct? looks like a lot of deletions

Reply:
> all these changes are correct? looks like a lot of deletions

yes. It's a little annoying in the diff because `incomplete` and `notApplicable` switched places in the artifact, but it reads better in the gatherer in that order, and I figured preserving `artifacts.json` order shouldn't negate that

Reply:
oh, and lots of deletions because all we offer for `notApplicable` is the `id`:

https://github.com/GoogleChrome/lighthouse/blob/11d33bb24317fb28693952e1f8898549154cfa58/types/artifacts.d.ts#L182

so there's no need to keep the rest around